src/registers/hcr_el2.rs

```diff
@@ -18,6 +19,53 @@ use tock_registers::{
 
 register_bitfields! {u64,
     pub HCR_EL2 [
+        ///
```

Comment:
```suggestion
```

Reply:
This is supposed to mean delete line. Unsure if I did that correctly.

Reply:
Done 👍🏻